components/dashboard/src/teams/sso/OIDCClientConfigModal.tsx

```diff
@@ -55,17 +60,21 @@ export const OIDCClientConfigModal: FC<Props> = ({ clientConfig, onClose }) => {
             <ModalBody>
                 {clientConfig?.active && (
                     <Alert type="warning" className="mb-4">
-                        Warning, you are editing an active SSO configuration. A misconfiguration may prevent any user
-                        from logging in.
+                        This is an active SSO configuration. In order to make changes, please create a new
+                        configuration. You may then verify and activate it.
```

Comment:
**nitpick**: Some minor copy suggestion—please pick up anything that looks better.

```suggestion
                        This configuration is currently active for single sign-on (SSO). To make any modifications, please 
                        create a new configuration. Once created, you can proceed to verify and activate it.
```